awx/api/serializers.py

```diff
@@ -140,6 +142,8 @@
     'credential_type': DEFAULT_SUMMARY_FIELDS,
 }
 
+import sdb
```

Comment:
you need to remove this or comment this out in order to install a prod instance because this is only available in the development environment